src/components/tooltip/tooltip.module.scss

```
@@ -12,7 +12,16 @@
   width: max-content;
   z-index: 1;
 
-  .triangle {
+  .triangle_bottom {
+    border-color: $tooltip-background-color transparent transparent transparent;
+    border-style: solid;
+    border-width: 5px;
+    position: absolute;
+    margin-right: var(--tooltip-margin-right);
+    top: 100%;
+    right: 50%;
+  }
+  .triangle_left {
```

```suggestion
  .triangleLeft {
```
As mentioned in other comment, nit, but classes use camelCase by convention in this repo